lib/__tests__/created_message_queue_handler.test.ts

```diff
@@ -147,6 +149,8 @@ const anAttachmentMeta = {
   media: "media.json"
 };
 
+const aUrl = "http://aUrl.com" as WebUrl;
```

Comment:
`HttpUrl` would be more correct since we're restricting the scheme

Reply:
or perhaps just URL ?

Reply:
cannot be "URL" as it conflicts with nodejs URL type

Reply:
can't we use that instead of creating a new type?

Reply:
probably we can. I guess that to that we must add the type to the api definition template (nunjucks) as we cannot directly reference a "native" type in the api specs. maybe with another pr ?